scripts/deploy-goerli.sh

```diff
@@ -5,6 +5,7 @@ set -eo pipefail
 source "${BASH_SOURCE%/*}/common.sh"
 
 [[ "$ETH_RPC_URL" && "$(seth chain)" == "goerli" ]] || die "Please set a goerli ETH_RPC_URL"
+[[ "$RWA_GEM_LIMIT" ]] || die "Please set RWA_GEM_LIMIT"
```

Comment:
I believe `RWA_URN_GEM_LIMIT` is a better name to convey what it means.
We should make it explicit that this is a requirement of the urn.

Reply:
Agree 👍

Reply:
fixed